packages/paste-core/components/card/stories/index.stories.tsx

```diff
@@ -1,10 +1,12 @@
 import * as React from 'react';
 import {Heading} from '@twilio-paste/heading';
 import {withKnobs, select} from '@storybook/addon-knobs';
+import {Box} from '@twilio-paste/box';
+import {Paragraph} from '@twilio-paste/paragraph';
 import {Text} from '@twilio-paste/text';
-import {Padding as PaddingProps} from '@twilio-paste/style-props';
+import type {Padding as PaddingProps} from '@twilio-paste/style-props';
```

Comment:
Good question. I think it was done this way because of the story also named Padding. These are pretty old stories, so I'll just rename that one, and use the type Padding properly here.